bisq-easy/src/main/java/bisq/bisq_easy/BisqEasyNotificationsService.java

```diff
@@ -144,11 +182,61 @@ public void dismissNotification() {
         updateNotificationVisibilityState();
     }
 
+    private void handleNotification(@Nullable ChatNotification notification) {
+        if (notification == null) {
+            return;
+        }
+        scheduleRefresh();
+    }
+
+    private void scheduleRefresh() {
+        long currentTime = System.currentTimeMillis();
+        synchronized (refreshLock) {
+            if (lastRefreshTime > 0 && currentTime - lastRefreshTime > MAX_WAIT_TIME_MS) {
+                if (scheduledRefresh != null) {
+                    scheduledRefresh.cancel(false);
+                    scheduledRefresh = null;
+                }
+                scheduler.execute(this::executeRefresh);
+                return;
+            }
+
+            if (scheduledRefresh != null) {
+                scheduledRefresh.cancel(false);
+            }
+
+            scheduledRefresh = scheduler.schedule(this::executeRefresh,
+                    DEBOUNCE_DELAY_MS, TimeUnit.MILLISECONDS);
+        }
+    }
+
+    private void executeRefresh() {
+        synchronized (refreshLock) {
+            lastRefreshTime = System.currentTimeMillis();
+            scheduledRefresh = null;
+        }
+        refreshNotifications();
+    }
+
+    private void refreshNotifications() {
+        Set<ChatNotification> validNotifications = chatNotificationService.getNotConsumedNotifications(ChatChannelDomain.BISQ_EASY_OPEN_TRADES)
```

Comment:
We should extract the notifications which do not have a trade or tradeChannel and put them into a queue which gets reprocessed when there is a trade or tradeChannel update.

I can be that we get a notification earlier than the trade message, thus we would drop the notification. With putting in  into a queue and reprocess it once the trade or channel gets created we will apply it correctly.